cockatrice/src/remoteclient.cpp

```diff
@@ -41,8 +41,13 @@ RemoteClient::RemoteClient(QObject *parent)
     socket->setSocketOption(QAbstractSocket::LowDelayOption, 1);
     connect(socket, SIGNAL(connected()), this, SLOT(slotConnected()));
     connect(socket, SIGNAL(readyRead()), this, SLOT(readData()));
+
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
+    connect(socket, &QTcpSocket::errorOccurred, this, &RemoteClient::slotSocketError);
+#else
     connect(socket, SIGNAL(error(QAbstractSocket::SocketError)), this,
```

Comment:
`QObject::connect: No such signal QProcess::error(QAbstractSocket::SocketError)` did this change?

Reply:
error signal became errorOccurred in 5.15+